speculation-rules.bs

```diff
@@ -805,7 +805,16 @@ While efforts have been made to minimize the privacy impact of prefetching, some
 
 Some user agents <a href="https://privacycg.github.io/storage-partitioning/">partition storage</a> according to the site or origin of the top-level document. In order for prefetching and prerendering to be useful, it is therefore essential that prefetching or prerendering of a document either occur in the partition in which the navigation would occur (e.g., for a same-site URL) or in an isolated partition, so as to ensure that prefetching does not become a mechanism for bypassing the partitioning scheme.
 
-<div class="issue">Expand this section once more detail on prefetch and prerender partitioning mechanism is specified.</div>
+The <a href="prefetch.html">prefetch specification</a> allows issuing HTTP requests which behave consistently with the partitioning scheme. If a navigation using its response would load a document in the same partition (approximately, the top-level site would not change), then partitioned state (e.g., cookies) can be sent, as they can with subresource requests and scripted fetches. If it would load a document in another partition, it would be inconsistent with the partitioning scheme to use partitioned state for the destination partition (since this would cross the boundary between partitions without a top-level navigation) and also inconsistent to use partitioned state within the originating partition (since this would result in the user seeing a document with different state than a non-prefetched navigation). Instead, a third, initially empty, partition is used for such requests. These requests therefore contain no partitioned state from either partition (though it may still be possible to fingerprint the user agent by other means).
+
+A notable exception to this policy is that in order to determine if using an empty partition affected the resource returned by a cross-partition request, one of two conditions have to be met:
```

Comment:
This long sentence is the most unclear thing here. I'm not clear how one of these conditions being met constitutes an exception to the policy stated above.

This _might_ be a correct rephrasing?

> Using this third empty partition will only give correct results if one of the following conditions is met:
> 
> 1. the response declares that the resource fetched with no credentials can be used, even if a normal (non-prefetch) navigation would have been done with credentials; or
> 2. the destination partition contains no credentials.
> 
> We can only determine if the latter condition is met by examining the destination partition's state during the prefetch process. To avoid this...

Reply:
I'm trying to call extra attention to the fact that otherwise we can simply say "we never examine cross-partition state", but in this case we do (cautiously).